### DISCUSSION TITLE
使用QtCreator编译失败

### DISCUSSION THREAD

**Discussion by behtgod** [CefView/QCefView#168] [Q&A]
我用Qt6.3.1，打开cmake工程进行编译，会提示

>mt.exe : general error c101008c: Failed to read the manifest from the resource of file "D:/workspace/opensource/build-QCefView->Desktop_Qt_6_3_1_MSVC2019_64bit-Release/CefViewCore/output/Release/bin/CefViewWing.exe". 指定的映像文件不包含资源区域。

请问有没有人遇到过这个问题？

**Comment by njuzhou** [comment]
往前翻一下，有我相同的提问和回答

**Reply by behtgod** [reply]
有没有人知道为什么呢？
